Not working with v-leaflet 0.6.3

When I try to use v-leaflet-heat 0.2 with v-leaflet 0.6.3 the vaadin compiler gives these errors:

```
[INFO] Using com.vaadin:vaadin-client-compiler version 7.5.4
[INFO] Compiling module de.vegetweb.flora_web.vaadin.ExampleAppWidgetset
[INFO]    Tracing compile failure path for type 'org.vaadin.addon.leafletheat.client.LHeatLayerConnector'
[INFO]       [ERROR] Errors in 'jar:file:/home/dve/.m2/repository/org/vaadin/addon/v-leaflet-heat/0.2/v-leaflet-heat-0.2.jar!/org/vaadin/addon/leafletheat/client/LHeatLayerConnector.java'
[INFO]          [ERROR] Line 24: Cannot cast from AbstractLeafletComponentState to LHeatLayerState
[INFO]          [ERROR] Line 23: The return type is incompatible with AbstractLeafletLayerConnector<AbstractJsonOptions>.getState()
[INFO]       [ERROR] Errors in 'jar:file:/home/dve/.m2/repository/org/vaadin/addon/v-leaflet-heat/0.2/v-leaflet-heat-0.2.jar!/org/vaadin/addon/leafletheat/client/LHeatLayerState.java'
[INFO]          [ERROR] Line 5: AbstractLeafletComponentState cannot be resolved to a type
[INFO]          [ERROR] Line 3: The import org.vaadin.addon.leaflet.client.AbstractLeafletComponentState cannot be resolved
[INFO]    Tracing compile failure path for type 'org.vaadin.addon.leafletheat.client.LHeatLayerState'
[INFO]       [ERROR] Errors in 'jar:file:/home/dve/.m2/repository/org/vaadin/addon/v-leaflet-heat/0.2/v-leaflet-heat-0.2.jar!/org/vaadin/addon/leafletheat/client/LHeatLayerState.java'
[INFO]          [ERROR] Line 5: AbstractLeafletComponentState cannot be resolved to a type
[INFO]          [ERROR] Line 3: The import org.vaadin.addon.leaflet.client.AbstractLeafletComponentState cannot be resolved
[INFO]    [ERROR] Aborting compile due to errors in some input files
```

--- a/pom.xml
+++ b/pom.xml
@@ -298,7 +298,7 @@
         <dependency>
             <groupId>org.vaadin.addon</groupId>
             <artifactId>v-leaflet</artifactId>
-            <version>0.6.1</version>
+            <version>0.6.3</version>
         </dependency>
         <dependency>
             <groupId>org.peimari</groupId>

--- a/src/main/java/org/vaadin/addon/leafletheat/client/LHeatLayerState.java
+++ b/src/main/java/org/vaadin/addon/leafletheat/client/LHeatLayerState.java
@@ -1,6 +1,6 @@
 package org.vaadin.addon.leafletheat.client;
 
-import org.vaadin.addon.leaflet.client.AbstractLeafletComponentState;
+import org.vaadin.addon.leaflet.shared.AbstractLeafletComponentState;
 
 public class LHeatLayerState extends AbstractLeafletComponentState {
     public String latlngJson;

